testing/tests/DevExpress.exporter/jspdfParts/v3/jspdf_v3.dataGrid.tests.js

```diff
@@ -118,6 +121,9 @@ function createMockPdfDoc() {
         if(arguments.length >= 3 && arguments[3].baseline === 'alphabetic') {
             arguments[3] = undefined;
         }
+        if(this.__logOptions.textOptions.hAlign !== true && arguments.length >= 3 && isDefined(arguments[3]) && arguments[3].align === 'left') {
```

Comment:
this.__logOptions.textOptions.hAlign -> this.__logOptions?.textOptions?.hAlign ?

Reply:
test sources are not transpiled, we can keep this line until the first nullref issue